examples/transient_adiabatic_ce/pin-td.py

```diff
@@ -0,0 +1,128 @@
+
+# Instantiate an EnergyGroups object for one group data
+one_group = openmc.mgxs.EnergyGroups()
+one_group.group_edges = [fine_groups.group_edges[0], fine_groups.group_edges[-1]]
+
+# Instantiate a clock object
+t_outer = np.arange(0., 1.0, 5.e-1)
+clock = openmc.kinetics.Clock(start=0., end=0.5, dt_inner=1.e-2, t_outer=t_outer)
+
+# Prescribe the transient as a dictionary of densities and temperatures 
+Transient = {}
+
+# Create entries for each material 
+for material in materials_file:
+    MatChange = {
+            material.name: {},
+            }
+    Transient.update(MatChange)
+    for t in t_outer:
+        TimeDict = {
+                t: {
+                    'density' : {},
+                    'temperature' : {},
+                    }
+                }
+        Transient[material.name].update(TimeDict)
+
+# Fill the entries with the desired values
+for material in materials_file:
+    if material.name == 'Moderator':
+        Transient[material.name][0]['density'] = material.density
+        Transient[material.name][0.5]['density'] = material.density*0.9
+        for t in t_outer:
+            Transient[material.name][t]['temperature'] = material.temperature
+    else:
+        for t in t_outer:
+            Transient[material.name][t]['density'] = material.density
+            Transient[material.name][t]['temperature'] = material.temperature
+
+# Instantiate a kinetics solver object
+solver = openmc.kinetics.Solver(directory='PIN_TD')
+solver.num_delayed_groups           = 6
+solver.amplitude_mesh               = full_pin_cell_mesh
+solver.shape_mesh                   = full_pin_cell_mesh
+solver.tally_mesh                   = full_pin_cell_mesh
+solver.one_group                    = one_group
+solver.energy_groups                = energy_groups
+solver.fine_groups                  = fine_groups
+solver.tally_groups                 = energy_groups
+solver.geometry                     = geometry
+solver.settings_file                = settings_file
+solver.materials_file               = materials_file
+solver.transient                    = Transient
+solver.inner_tolerance              = np.inf
+solver.outer_tolerance              = np.inf
+solver.method                       = 'ADIABATIC'
+solver.multi_group                  = False
+solver.clock                        = clock
+solver.mpi_procs                    = 1
+solver.threads                      = 1
+solver.core_volume                  = np.pi*0.39218**2*365.76
+solver.constant_seed                = True
+solver.seed                         = 1
+solver.min_outer_iters              = 1
+solver.use_pcmfd                    = False
+solver.use_agd                      = False
+solver.condense_dif_coef            = True
+solver.chi_delayed_by_delayed_group = True
+solver.chi_delayed_by_mesh          = False
+solver.use_pregenerated_sps         = False
+solver.run_on_cluster               = False
+solver.log_file_name                = 'log_file.h5'
```

Comment:
It seems this is using the default value, maybe we should remove the lines here which are just using the default value from the solver interface.

Reply:
It's just an example, it's not slowing anything down. I think it's good to have the example file show all the options the user has.

Reply:
Very reasonable point!